### PR TITLE
Add Audit Audience Mapper

### PR DESCRIPTION
**Pull request by tomhollingworth**
### Add
- Add Audit Audience Mapper for Audit trail permissions
- Add additional clients to add custom client scopes to

### Fix
- Fix ClientScope styling to match others

### CODE REVIEW

File: content/deploy/install/services.md [hunk 1]
```diff
@@ -317,6 +317,35 @@ alpha:
     replicas: 1
 ```
 
+### Enable Audit Subgraph
+
+The Audit trail subgraph is super composed into the router in order to use. To enable router to use and compose the subgraph:
```
Comment (MattDodsonEnglish) [inline content/deploy/install/services.md:322]: @tomhollingworth Is this about these features from Apollo Router:
-[ Rover SuperGraph ](https://www.apollographql.com/docs/rover/commands/supergraphs/)
- [Implementing a  Subgraph](https://www.apollographql.com/docs/apollo-server/using-federation/apollo-subgraph-setup)

And what does the term "super compose" mean? My preliminary internet searching turns up a lot of false positives.


Just yesterday, we pushed a doc about using the Audit trail: https://docs.rhize.com/how-to/audit/

I might want to add some context at that doc, or at least in the glossary. Maybe we can make a conceptual doc about this later.

But at least we can link and define the terms for now.

File: content/deploy/install/services.md [hunk 2]
```diff
@@ -317,6 +317,35 @@ alpha:
     replicas: 1
 ```
 
+### Enable Audit Subgraph
```
Comment (MattDodsonEnglish) [inline content/deploy/install/services.md:320]: If `subgraph` will be a common term going forward, we also need to add the term it to `accept.txt`. https://github.com/libremfg/libremfg.github.io/blob/main/.github/styles/Vocab/docs/accept.txt 

I can do that. My bad, the prose-checker is a completely undocumented CI action that I just added in #46.

File: content/deploy/install/services.md (hunk 1 above)
Comment (tomhollingworth) [reply content/deploy/install/services.md:322]: Ah yeah i was using a bit of slang here, by "super compose" I meant to compose the supergraph from multiple subgraphs. This should be re-written to 

| The Audit trail subgraph is composed in the Router's Supergraph in order to use in the UI. To enable router to introspect and compose the Audit trail subgraph into the supergraph:

Concepts:

Our microservices, Baas, Core, BPMN and Audit all expose a GraphQL API for their data/functions. We have a script that runs when Router starts up, to (introspect)[https://graphql.org/learn/introspection/] microservices, and compose all the subgraphs into a supergraph. It does this by stitching all the subgraph schemas together, linking/extending identical objects, and removing what isn't common. Once this composition step is complete, it then serves up the supergraph as a single GraphQL API. When receiving a request, it reverses which parts of the query need to go to which microservice and joins the resulting data structures.

File: content/deploy/install/services.md (hunk 2 above)
Comment (tomhollingworth) [reply content/deploy/install/services.md:320]: Yeah, we will need something in "Concepts" for Subgraph / Supergraph

File: content/deploy/install/services.md (hunk 1 above)
Comment (MattDodsonEnglish) [inline content/deploy/install/services.md:322]:
```suggestion
To use the Audit trail in the UI, you must add the Audit trail subgraph into the router. To enable router to use and compose the subgraph:
```

**Comment by MattDodsonEnglish** [comment]
Thanks for the update! I'm going to merge now and keep the concept as a draft. But I'll rework it and make it live in the coming weeks, as we discussed.

**Pull request merged**